scripts/imgtool/image.py

```diff
@@ -612,6 +647,21 @@ def create(self, key, public_key_format, enckey, dependencies=None,
             if compression_tlvs is not None:
                 for tag, value in compression_tlvs.items():
                     prot_tlv.add(tag, value)
+
+            if self.vid is not None:
+                vid = parse_uuid(uuid.NAMESPACE_DNS, self.vid)
```

Comment:
why is DNS used? What other options are there instead of DNS?

Reply:
Because it is the simplest thing with a registry to use to specify the vendor.
The python package includes the following namespaces:
 - DNS: When this namespace is specified, the name string is a fully qualified domain name.
 - URL: When this namespace is specified, the name string is a URL.
 - OID: When this namespace is specified, the name string is an ISO OID.
 - X500: When this namespace is specified, the name string is an X.500 DN in DER or a text output format.

It also aligns with the values used in SUIT:

> The RECOMMENDED method to create a vendor ID is:
> The "IANA UUID Namespace ID for DNS" is:
> 6ba7b810-9dad-11d1-80b4-00c04fd430c8
> Vendor ID = UUID5(<IANA UUID Namespace ID DNS>, vendor domain name)